### PR TITLE
Add badges into README about license and packagist

### PR DESCRIPTION
**Pull request by matks**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | 

#### Summary

This PR adds multiple badges into the README. Badges are provided by http://shields.io/ and rely on data sources provided by GitHub and Packagist

Badges are really beautiful 🤩 but also can provide useful informations for people quickly browsing GitHub repositories.

### CODE REVIEW

**Comment by greg0ire** [comment]
Let's retarget this to the lowest maintained branch in case [we change the default branch](https://github.com/orgs/doctrine/teams/doctrinecore/discussions/19)

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Doctrine Migrations
 
 [![Build Status](https://travis-ci.org/doctrine/migrations.svg)](https://travis-ci.org/doctrine/migrations)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/doctrine/migrations/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/doctrine/migrations/?branch=master)
-[![Code Coverage](https://scrutinizer-ci.com/g/doctrine/migrations/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/doctrine/migrations/?branch=master)
+[![Code Coverage](https://codecov.io/gh/doctrine/migrations/branch/master/graph/badge.svg)](https://codecov.io/gh/doctrine/migrations/branch/master)
```
Comment (greg0ire) [inline README.md:4]: My rebase also brought this badge-related change I did against 3.0.x, which is good to also backport here.

**Comment by matks** [comment]
> Let's retarget this to the lowest maintained branch in case [we change the default branch](https://github.com/orgs/doctrine/teams/doctrinecore/discussions/19)

I think I can guess what discussion this is about 😛 however FYI this link is private, I cant see it 😄 it's probably for maintainers only

File: README.md (hunk 1 above)
Comment (matks) [reply README.md:4]: Indeed 👍

**Comment by matks** [comment]
Thank you for
- very fast answer
- doing the retarget + rebase yourself (I could have done it if you asked me to 😉 )

Amazing work you're doing it here, on Sunday evening ❤️

**Comment by greg0ire** [comment]
> I think I can guess what discussion this is about stuck_out_tongue however FYI this link is private, I cant see it smile it's probably for maintainers only

Oh indeed, sorry, I will know better from now on!

**Pull request merged**